data/language/en-GB.txt

```diff
@@ -2169,7 +2169,6 @@ STR_3136    :Warning: This design will be built with an alternative vehicle type
 STR_3137    :Select Nearby Scenery
 STR_3138    :Reset Selection
 STR_3139    :Cable lift unable to work in this operating mode
-STR_3140    :Cable lift hill must start immediately after station
```

Comment:
It doesn't matter at this point, but any particular reason for removing the existing string, rather than updating it?

Reply:
Seeing how long old translations linger (e.g. when the Russian translation was updated recently) I figured it would be preferable to have the correct string show up in English, rather than an inaccurate translated one.